yt/data_objects/data_containers.py

```diff
@@ -207,6 +207,11 @@ def _set_center(self, center):
                 self.center = self.ds.find_min(center[4:])[1]
         else:
             self.center = self.ds.arr(center, "code_length", dtype="float64")
+
+        if self.center.ndim > 1:
+            mylog.warning("Removing singleton dimensions from 'center'.")
+            self.center = np.squeeze(self.center)
+
```

Comment:
while you're at it, I'd suggest adding an error here in case the `self.center.ndim > 1` still holds. It doesn't make sense to pass an array with shape say, (3,3), but it could happen accidentally, so better catch the error as early as possible :-)

Reply:
yes, good idea! will add it.